esp/mlrs-wifi-bridge/mlrs-wifi-bridge.ino

```diff
@@ -133,7 +140,7 @@ void setup()
     size_t rxbufsize = SERIAL.setRxBufferSize(2*1024); // must come before uart started, retuns 0 if it fails
     size_t txbufsize = SERIAL.setTxBufferSize(512); // must come before uart started, retuns 0 if it fails
 #ifdef SERIAL_RXD // if SERIAL_TXD is not defined the compiler will complain, so all good
-  #ifdef SERIAL_INVERT
+  #ifdef USE_SERIAL_INVERTED
```

Comment:
I need to think about this, but for the moment ok
will not compile, right, since SERIAL_INVERT isn't always defined

Reply:
It does compile, please check.

Reply:
I'm just looking at the code, but could it do the correct thing when SERIAL_INVERT is not defined? (which it seems to me it isn't). Maybe it compiles because that field has a default?

Reply:
https://github.com/rotorman/mLRS/blob/9c5891f0937efc8e606da33ffba4c40e9eb97336/esp/mlrs-wifi-bridge/mlrs-wifi-bridge.ino#L144-L152

Reply:
Ah darn, now I get it, my bad.

Reply:
:)

Reply:
Fixed now, sorry my bad.